edit-post/keyboard-shortcuts.js

```diff
@@ -5,7 +5,11 @@ import { rawShortcut, displayShortcut } from '@wordpress/keycodes';
 
 export default {
 	toggleEditorMode: {
-		value: rawShortcut.secondary( 'm' ),
-		label: displayShortcut.secondary( 'm' ),
+		raw: rawShortcut.secondary( 'm' ),
+		display: displayShortcut.secondary( 'm' ),
+	},
+	toggleSidebar: {
+		raw: rawShortcut.primaryShift( ',' ),
```

Comment:
This shortcut is opening a new tab in chrome (Chrome support page)

Reply:
I wonder is this is another keyboard layout issue, as this key combination has no effect on my keyboard (cmd+shift+,).

Reply:
Probably, I have a french Keyboard

Reply:
It looks like there's a `,` key and you hold shift to output `?` on a French keyboard. On a British keyboard layout it's `,` and you hold shift to output `<`.

However, on a British keyboard layout, when you press `cmd+shift+,`, the event handler returns that the `,` was pressed. On the French keyboard layout it returns `?`.

That seems completely inconsistent, and I don't know why that's the case.